### ISSUE TITLE
  Screening or "knocking" for Unapproved Callers (Moderator video/audio preview)

### ISSUE THREAD

**Issue by heyakyra** [vector-im/element-meta#1081]
### Your use case

tl;dr: "knocking" for video calls, where users wait in the lobby to be approved

Originally filed at https://github.com/vector-im/element-call/issues/174 but probably more appropriate here

**Is your feature request related to a problem? Please describe.**
* As a teacher, I want to be able to conduct a lesson where my students join to watch/listen to a lecture while participating in text discussion, but prevent students from presenting their own cam/mic to the rest of the class without approval (I should be able to see their stream before approving it).
* As a consultant, I want to be able to facilitate a public panel discussion with my peers for a medium sized audience and moderate the participants as they join so that I don't have to create an invite-only event which would limit participation. 
* As a member of a video mentorship community, I want to facilitate a safe space where new participants below a certain power level can join without fear that they will disrupt the community members already in a call, but still benefit from the ongoing conversation/tutoring happening via video.

**Describe the solution you'd like**
The ability to set a minimum power level to join a call—either at all, or without screening by users at another specified power level—but still be able to view the call and participate in text-only conversation beforehand. In each of the above scenarios, all other participants are by default just allowed to join in view-only mode as described in vector-im/element-web#223 

~Also, a video-focused layout (a la Jitsi/8x8, Zoom, Google Meet, etc) would be nice, where text is secondary for a specific room.~ (Done!) https://github.com/matrix-org/matrix-spec/issues/730

Related: https://github.com/vector-im/element-meta/issues/1476

### Have you considered any alternatives?

* Using another platform ):

### Additional context

Unsure, happy to provide more examples.
